### ISSUE TITLE
version selector is flipping out

### ISSUE THREAD

**Issue by nornagon** [electron/fiddle#1145]
Electron Fiddle Version 0.29.1 (17.4.1)

![Screen Shot 2022-08-03 at 2 47 58 PM](https://user-images.githubusercontent.com/172800/182718498-e65b04c9-f926-4e6b-8228-35a4eb12766d.png)

Not sure what causes it. Some stuff works fine:
![Screen Shot 2022-08-03 at 2 48 55 PM](https://user-images.githubusercontent.com/172800/182718588-dc1249ce-9f6c-4186-ab63-4559dc89a652.png)

other stuff is completely broken:
![Screen Shot 2022-08-03 at 2 49 14 PM](https://user-images.githubusercontent.com/172800/182718625-7d8c13f2-fc3b-4e4a-9499-807ac7aa089e.png)

**Comment by nornagon** [comment]
Probably something to do with this?

![Screen Shot 2022-08-03 at 2 50 36 PM](https://user-images.githubusercontent.com/172800/182718802-6dee8dc3-37d2-4dd9-bee5-5199dfcbb5c4.png)

**Comment by codebytere** [comment]
Hmm - i'm not seeing anything like that 🤔 

<img width="619" alt="Screen Shot 2022-08-04 at 9 38 49 AM" src="https://user-images.githubusercontent.com/2036040/182791385-29c7ea53-3eaf-4286-8a1a-1375bbc7c185.png">
<img width="604" alt="Screen Shot 2022-08-04 at 9 38 58 AM" src="https://user-images.githubusercontent.com/2036040/182791376-467b85b8-f47b-43ca-90c0-eb832222a773.png">

**Comment by BlackHole1** [comment]
Everything works fine in my Windows virtual machine
<img width="523" alt="image" src="https://user-images.githubusercontent.com/8198408/182793037-b08e5366-7340-41f3-b18b-5bf225bb93b7.png">

**Comment by BlackHole1** [comment]
I have located the problem. code:

https://github.com/electron/fiddle/blob/d77d3ac461f695d9cfb03113a541aae1307df1b1/src/renderer/components/version-select.tsx#L191-L199

But it's a bit tricky to fix

**Comment by codebytere** [comment]
@BlackHole1 can you say more? it's not clear what "the problem" is here in terms of why the versions wouldn't show as available.

**Comment by BlackHole1** [comment]
> can you please say more? it's not clear what "the problem" is here in terms of why the versions wouldn't show as available since just showing the code doesn't really describe much :)

When `disableDownload` conditions are met, the problem is reproduced.

https://github.com/electron/fiddle/blob/d77d3ac461/src/utils/disable-download.ts#L12-L20

This method can be changed to the following code for replication:

```ts
export function disableDownload(version: string) {
  return version === '8.1.0'
}
```

<img width="1285" alt="image" src="https://user-images.githubusercontent.com/8198408/182809350-dfd754c0-be60-4e97-bcf3-c5e02e45313b.png">

From the errors reported, I have a feeling it's a problem with `@blueprintjs`.

I tried several fixes, but none of them worked. But when I removed `Tooltip`, the problem was solved.
